src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,74 @@
+import styled from '@emotion/styled';
+import checkmark from './checkmark.svg';
+import Label from '../Label';
+
+const StyledCheckbox = styled.input<CheckboxProps>`
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  -o-appearance: none;
+
+  ${({ theme, error }) => {
+    return {
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+      height: '22px',
+      width: '22px',
+      border: '2px solid',
+      borderColor: theme.semantic.color.border.default,
+      borderRadius: theme.semantic.border.radius.small,
```

Comment:
This may be a little harder to deconstruct since the similarities end at semantic, in my opinion but open to it